.github/workflows/build.yml

```
@@ -9,7 +9,7 @@ on:
     - development
   pull_request:
     branches:
-    - development
+    - **
```

actionlint (1.7.7)

could not parse as YAML: yaml: line 12: did not find expected alphabetic or numeric character

(syntax-check)